uaclient/entitlements/base.py

```diff
@@ -83,6 +87,14 @@ def help_info(self) -> str:
     def static_affordances(self) -> "Tuple[StaticAffordance, ...]":
         return ()
 
+    # A list of packages that are not compatible with the entitlement.
+    # When we are enabling the entitlement we can directly ask the user
+    # if those entitlements can be disabled before proceding.
+    # Overridden in livepatch and fips
```

Comment:
Let's make this comment into a proper docstring on the method instead of a leading comment.